lib/tasks/hathi.rake

```diff
@@ -11,7 +11,17 @@ conn = Faraday.new(url: bibdata_url) do |faraday|
 end
 
 namespace :hathi do
-  
+  desc "load institution-specific hathi access file into HathiAccess objects"
+  task load_access: :environment do
+    input_location = ENV['HATHI_INPUT']
+    origin = ENV['HATHI_ORIGIN']
+    abort "usage: HATHI_INPUT=/path/to/file.tsv HATHI_ORIGIN=CUL rake hathi:load_access" unless input_location && origin
+    before_count = HathiAccess.all.count
+    Hathi::LoadAccessFile.new(input_location: input_location, origin: origin).load
+    after_count = HathiAccess.all.count
+    puts "#{after_count - before_count} records created"
```

Comment:
This function won't be able to be run twice over the same file right? Since you call `create!` I'd expect it to make a bunch of records with duplicate information. Should we find_or_initialize or something similar?

Reply:
Updated to use `find_or_create_by!` does that seem good? Thanks for catching this.

Reply:
It does, unless the ALLOW or DENY status is likely to change? I'm okay with it for now, but I wonder if in the end we'll end up with find_or_initialize_by(oclc_number:, origin:), and then update it and save.

Reply:
Makes sense. I created https://github.com/pulibrary/marc_liberation/issues/763